src/Task.cc

```diff
@@ -2727,6 +2727,25 @@ void Task::open_mem_fd_if_needed() {
   }
 }
 
+bool Task::open_pagemap_fd() {
```

Comment:
Why are we exposing this?

Seems like we could just have `Task::pagemap_fd()` which opens the fd if needed and returns it.

Reply:
Was just a copy of *mem_fd*, I reduced it in V2.